### PR TITLE
Fixed logout

### LINKED ISSUE #63: Log Out not working in Report Page
### Problem

When a user is signed in and clicks on the Report page, the "Log out" option does not perform the intended task (/Places/complaints)

image

### Expected Behaviour

The "Log out" button should log the user out when clicked on the Report page or any other page within the application. The logout functionality is already implemented but does not work on this page.

Note:

- Create a new branch for every task to isolate changes
- Start working on the issue only when it is assigned to you
- Submit the PR on time, no extra time will be allotted

### PR DESCRIPTION
**Pull request by mangalgithub**
Fixed the logout button in report page

## Description

Added some bootstrap and jquery cdn to make the logout functional.

Fixes #63 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Motivation and Context

As logout button is not working in report page and it is necessary to get user logged out so that no one can access through there account.

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

File: Visitor/urls.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from django.urls import path
+from django.contrib import admin
```
Comment (kunal2812) [inline Visitor/urls.py:2]: Why is this needed ?

File: Visitor/views.py
Comment (kunal2812) [inline Visitor/views.py:1]: Remove changes from this file. Not needed for the fix.

File: static/css/styles.css [hunk 2]
```diff
@@ -1413,4 +1413,7 @@ header.masthead .masthead-heading {
   text-align: center;
   font-size: 0.9rem;
   font-family: "Montserrat", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+#navbarResponsive {
+  background-color: transparent;
```
Comment (kunal2812) [inline static/css/styles.css:1418]: Why ?

File: templates/Places/complaints.html [hunk 3]
```diff
@@ -89,14 +95,20 @@
         }
 
     </style>
+      <!-- Bootstrap  -->
+     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (kunal2812) [inline templates/Places/complaints.html:99]: Remove these. Are these scripts required ?

File: static/css/styles.css (hunk 2 above)
Comment (mangalgithub) [reply static/css/styles.css:1418]: yesterday what I have suggested due to that some I am facing gray navbar but now I cloned and restart without this line now it working fine.
So, no need of this. I am removing this.

File: templates/Places/complaints.html (hunk 3 above)
Comment (mangalgithub) [reply templates/Places/complaints.html:99]: Yes sir this is not needed I checked without this.

**Review by kunal2812 — APPROVED** [review]
